ksa_compliance/templates/e_invoice.xml

```diff
@@ -118,6 +120,14 @@
                 <cac:TaxScheme>
                     <cbc:ID>VAT</cbc:ID>
                 </cac:TaxScheme>
+            </cac:PartyTaxScheme>
+             {% else %}
```

Comment:
I don't understand this `else` block. Is this for `if buyer_details.party_identifications`? In which case, `first` would return none or throw an exception